services/Donkey/src/donkey/core.clj

```diff
@@ -219,13 +216,21 @@
    :art-id "donkey"
    :service "donkey"})
 
+(def exception-handlers
+  {:handlers {::ex/request-validation (as-de-exception-handler ex/request-validation-handler ec/ERR_ILLEGAL_ARGUMENT)
+              :invalid-configuration (as-de-exception-handler invalid-cfg-handler ec/ERR_CONFIG_INVALID)
+              ::ex/response-validation (as-de-exception-handler ex/response-validation-handler ec/ERR_SCHEMA_VALIDATION)
+              ::ex/default unchecked-handler}})
+
 (defn site-handler
   [routes-fn]
   (-> (delayed-handler routes-fn)
-    wrap-keyword-params
-    wrap-lcase-params
-    wrap-query-params
-    (tc/wrap-thread-context svc-info)))
+      (wrap-exceptions exception-handlers)
+      wrap-logging
+      wrap-user-info
+      wrap-keyword-params
+      wrap-lcase-params
+      wrap-query-params))
```

Comment:
Do we want to rearrange the order here? I would think we would want to wrap user-info before wrapping exceptions, before wrapping logging...
So
```clojure
(-> (delayed-handler routes-fn)
  wrap-keyword-params
  wrap-lcase-params
  wrap-query-params
  wrap-user-info
  (wrap-exceptions exception-handlers)
  wrap-logging))
```